### ISSUE TITLE
Clicking on "Show Details" for the Ontology-KP API registry item causes the item to disappear

### ISSUE THREAD

**Issue by bill-baumgartner** [SmartAPI/smartAPI#126]
When clicking on the "Show Details" button for the `Ontology-KP API` registry item I've noticed that instead of the API details being shown, the entire registry item disappears from the screen. I've tried this in both Chrome and Safari with similar results. Before & After screenshots attached in case they are helpful.  Thanks!

## Before
![before](https://user-images.githubusercontent.com/7217210/118004728-f108e600-b306-11eb-97b1-1abb590769f0.png)

## After
![after](https://user-images.githubusercontent.com/7217210/118004738-f49c6d00-b306-11eb-942b-b5f6abe3a584.png)

**Comment by marcodarko** [comment]
Hi @bill-baumgartner thanks for bringing this up to our attention, looks like the first endpoint specified is missing a "summary" or "description" field, so far all APIs provide that so our logic assumes that, we can adjust that on our next release but if you want a faster result we recommend you update your metadata and add this field.

**Comment by bill-baumgartner** [comment]
Hi @marcodarko, thanks for the quick reply! I'll add @balhoff to the thread as he is the owner of this service. Jim, I happened across this earlier this morning while looking for the query URL before you set me straight. Just fyi - looks like there may be a missing summary field in the Ontology KP SmartAPI registry entry.

**Comment by balhoff** [comment]
Thanks for looking into this, @bill-baumgartner!

@marcodarko I can't figure out what I'm missing in the metadata. Here is the one that has the weird disappearing behavior: https://stars-app.renci.org/sparql-kp/docs/docs.yaml

Here is another one I registered which I think has the same fields, and it works fine: https://stars-app.renci.org/cam-kp/docs/docs.yaml

**Comment by marcodarko** [comment]
@balhoff The GET endpoint for `/predicates` needs a summary like the POST method above it describing it.  Once that's done it should be all good

**Comment by balhoff** [comment]
I believe this is fixed now, thanks!

**Comment by marcodarko** [comment]
Thank you for updating your document! https://smart-api.info/registry?q=Ontology-KP%20API is working now.  We'll keep this open and to implement a more robust way to handle this kind of exceptions as it is not technically a required field.

**Comment by andrewsu** [comment]
I'm noticing this issue for the "Connections Hypothesis Provider API" as well (the first hit with this query https://smart-api.info/registry?q=hypothesis%20provider).  @marcodarko how can I find out who the owner of that registry entry is and/or where the metadata source URL lives?  (If easy, it would be worth scanning through all registry entries to find which others need a similar change...)

**Comment by marcodarko** [comment]
I see.. yeah it's the same issue, their /meta_knowledge_graph path doesn't have a summary. This particular one belongs to Luke Veenhuis [metaurl](https://raw.githubusercontent.com/di2ag/chp_metadata/master/smartapi.yaml) I'll adjust our frontend code to not require that and also try to find all the ones with this field missing.

**Comment by marcodarko** [comment]
["This API 'Object Resolution Service' ('0ecef0464ffbc5c31a402bf4d1a88395') belonging to 'mlev71' might require attention: 2 paths lack descriptions: /ark:/{ID}/doi:/{ID}", "This API 'Bio2RDF' ('fcd07ed906fb41c9eedc2406f65252f7') belonging to 'micheldumontier' might require attention: 2 paths lack descriptions: /{prefix}:{identifier}/sparql", "This API 'CGC Seven Bridges Datasets API' ('2796377f99440bf6a55f1ece09721cbd') belonging to 'nemanjavuk' might require attention: 7 paths lack descriptions: /datasets/{dataset_key}/v0/datasets/{dataset_key}/v0/{entity_key}/datasets/{dataset_key}/v0/{entity_key}/{entity_id}/datasets/{dataset_key}/v0/{entity1_key}/{entity_id}/{entity2_key}/datasets/{dataset_key}/v0/{entity_key}/schema/datasets/{dataset_key}/v0/query/datasets/{dataset_key}/v0/query/total", "This API 'CGC Seven Bridges Public API' ('1a23be8a57ccae5b524bcb639ff41028') belonging to 'nemanjavuk' might require attention: 37 paths lack descriptions: /users/{username}/billing/groups/{billing_group}/billing/groups/{billing_group}/breakdown/billing/invoices/{invoice_id}/projects/{owner}/projects/{owner}/{project}/projects/{owner}/{project}/files/projects/{owner}/{project}/tasks/projects/{owner}/{project}/apps/projects/{owner}/{project}/members/projects/{owner}/{project}/members/{username}/projects/{owner}/{project}/members/{username}/permissions/files/{file_id}/files/{file_id}/actions/copy/files/{file_id}/download_info/files/{file_id}/metadata/files/{file_id}/tags/upload/multipart/{upload_id}/upload/multipart/{upload_id}/complete/upload/multipart/{upload_id}/part/upload/multipart/{upload_id}/part/{part_number}/apps/{app_id}/apps/{app_id}/{app_id}/raw/apps/{app_id}/actions/copy/tasks/{task_id}/tasks/{task_id}/inputs/tasks/{task_id}/actions/abort/tasks/{task_id}/actions/run/tasks/{task_id}/execution_details/storage/volumes/{volume_id}/storage/volumes/{volume_id}/list/storage/volumes/{volume_id}/{object_id}/storage/volumes/{volume_id}/members/storage/volumes/{volume_id}/members/{username}/storage/volumes/{volume_id}/members/{username}/permissions/storage/imports/{import_id}/storage/exports/{export_id}", "This API 'FAIR Evaluator API' ('4831dbbe28707c16b8c2b513b3523402') belonging to 'markwilkinson' might require attention: 1 paths lack descriptions: /searches/{myid}", "This API 'Connections Hypothesis Provider API' ('855adaa128ce5aa58a091d99e520d396') belonging to 'veenhouse' might require attention: 1 paths lack descriptions: /meta_knowledge_graph"]

**Comment by marcodarko** [comment]
Turns out the issue was caused by a typo on the frontend, however only paths with a description or summary field will show up on the registry page (but will show up on the docs page). So with that in mind I checked the registry (for openapi apis only, v3) and these are the APIs that currently don't specify some sort of description on one or more paths.  So only 6 have this "issue" but doesn't break anything anymore it just gets ignored but if we want to let these owners know here are their apis-usernames.
